Test/comp/globalNamespace/globalNamespaceRunConfig.toml

```diff
@@ -0,0 +1,5 @@
+includes=["./globalNamespaceRun.dfy"]
+
+[options]
+outer-namespace=["my","happy"]
```

Comment:
This should be "my.happy" now, shouldn't it?

Reply:
Test no longer applies because you're can't specify --outer-namespace when doing `dafny run`. I know that MAYBE we want to enable this, but for now I won't since the use-case is underspecified.